tests/test_actions/test_watch.py

```diff
@@ -164,7 +177,7 @@ def test_watch_no_folders(self):
         watch_thread.daemon = True
         watch_thread.start()
         time.sleep(10) #Gives watch enough time to start up before creating the document
-        create_txt_file(file_name2, self.dir_name)#create a file to add and be ignored
+        create_txt_file(file_name2, self.dir_name)#create a file to add and be ignored -- why exactly should it be ignored if we pass in () as ignore list
```

Comment:
Is "why exactly should it be ignored if we pass in () as ignore list" a question or an explanation?

Reply:
Was just a note to myself for if we ever get around to implementing this testing